va/va_dec_av1.h

```diff
@@ -339,12 +339,17 @@ typedef struct  _VADecPictureParameterBufferAV1
 
     /** \brief Surface indices of reference frames in DPB.
      *
-     *  Contains a list of uncompressed frame buffer surface indices as references.
-     *  Application needs to make sure all the entries point to valid frames
-     *  except for intra frames by checking ref_frame_id[]. If missing frame
-     *  is identified, application may choose to perform error recovery by
-     *  pointing problematic index to an alternative frame buffer.
-     *  Driver is not responsible to validate reference frames' id.
+     *  Contains a list of uncompressed frame buffer surface indices as
+     *  references for the current frame. If the current frame is not an intra
```

Comment:
“for the current frame” is not accurate, these surfaces are not only used for the current frame, but also for future frames. Maybe some surface is not used for current frame, but used for future frame.